docs/source/tutorial/interfaces.rst

```diff
@@ -301,7 +299,7 @@ We can also define an implementation for ``Maybe``, as follows:
         Nothing  >>= k = Nothing
         (Just x) >>= k = k x
 
-Using this we can, for example, define a function which adds two
+Using this, we can, for example, define a function that adds two
```

Comment:
This is a British vs American English thing, and given that Idris is a Scottish programming language, we tend to use the former.

Reply:
The comma here would also be more common in British English, although it's true it is more easily omitted in British English.

Reply:
I removed the commas around "for example" to make it less choppy.

Concerning "that" vs "which", I guess it's more so a Scottish English thing (as less informal British English still favors "that" in restrictive clauses). Should I go ahead and revert the remaining "which"->"that"s which I haven't addressed yet?